iina/pt-BR.lproj/Localizable.strings

```diff
@@ -164,7 +164,10 @@
 "pl_menu.add_file" = "Adicionar Arquivo...";
 "pl_menu.add_sub" = "Adicionar Legenda…";
 "pl_menu.add_url" = "Adicionar URL...";
+"pl_menu.browser" = "Abrir no  navegador";
```

Comment:
Did you intentionally put two spaces there?

Reply:
I don't. I'll fix